src/constants.ts

```diff
@@ -1 +1,3 @@
 export const APP_NAME = "strike-off-objections-web";
+export const OBJECTIONS_SESSION = "objections_session";
```

Comment:
can you name it strike_off_objections_session as this means it won't get confused if other objections services are created in the future within Redis
eg . export const OBJECTIONS_SESSION_NAME = "strike_off_objections_session";

Reply:
Changed the name